schema/project-level/project-schema.json

```diff
@@ -626,8 +640,10 @@
                 "type": [
                   "number",
                   "array"
-                ]
-              }
+                ],
+                "uniqueItems": true
```

Comment:
We might want to allow coordinates to repeat (notably, to close a polygon).

Reply:
Ah yes, in fact I think `uniqueItems` needs to be removed at both levels, since at the first level a simple point geometry could have the same latitude and longitude.

I've pushed a commit to allow both.